install/generator/04-syndesis-db.yml.mustache

```diff
@@ -64,6 +64,71 @@
       #!/bin/bash
       /var/lib/pgsql/sampledb/add-sample-db.sh &>  /proc/1/fd/1
 
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+      syndesis.io/component: syndesis-db
+    name: syndesis-db-maintenance-script
+  data:
+    maintenance.sh: |
+      #!/bin/bash
+      cat << EOF| psql -h syndesis-db -U $POSTGRESQL_USER -d $POSTGRESQL_DATABASE
+      VACUUM FULL ANALYSE jsondb;
+      REINDEX TABLE jsondb;
+      SELECT COUNT(*) FROM jsondb;
+      EOF
+
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: syndesis-db-maintenance
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+      syndesis.io/component: syndesis-db-maintenance
+  spec:
+    schedule: "15 4 * * *"
+    jobTemplate:
+      spec:
+        template:
+          metadata:
+            labels:
+              syndesis.io/app: syndesis
+              syndesis.io/type: infrastructure
+              syndesis.io/component: syndesis-db-maintenance
+          spec:
+            containers:
+            - name: syndesis-db-maintenance
+              env:
+              - name: POSTGRESQL_USER
+                value: ${POSTGRESQL_USER}
+              - name: PGPASSWORD
+                value: ${POSTGRESQL_PASSWORD}
+              - name: POSTGRESQL_DATABASE
+                value: ${POSTGRESQL_DATABASE}
+              image: centos/postgresql-95-centos7
+              command:
+                - /bin/sh
+                - -c
+                - /data/maintenance.sh
+              volumeMounts:
+              - mountPath: /data
+                name: syndesis-db-maintenance-script
+            restartPolicy: Never
+            volumes:
+            - name: syndesis-db-data
+              persistentVolumeClaim:
+                claimName: syndesis-db
```

Comment:
@zregvart In certain setups (e.g. OpenShift Starter), it's forbidden to use the same PVC in different pods. 

This might be an issue e.g. on our dedicated cluster.

Actually we do this for our update pod, too, so that might be issue over there as well.